package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "javascript-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "extra": "babel-node extra"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/AbhishekPunj0812/javascript-server.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/AbhishekPunj0812/javascript-server/issues"
+  },
+  "homepage": "https://github.com/AbhishekPunj0812/javascript-server#readme",
+  "dependencies": {
+    "@babel/core": "^7.12.3",
+    "@babel/node": "^7.12.1",
+    "@babel/preset-env": "^7.12.1",
+    "type": "module"
```

Comment:
This should not be in dependencies

Reply:
Sir I have a question that where are these instructions supposed to be. I'm unable to find sir.

Reply:
These are kind of config values about your project, and it vary depending on the project type and packages/ tools you are using. Usually when you install a package, In it's documention/ read.ME  file you will get all these info.
You can check this [link](https://nodejs.dev/learn/the-package-json-guide) for more info on package.json

Reply:
Sir have I made the right correction?

Reply:
I think you did not get my point here. In "dependencies" and "devDepenencies" we add the project dependancy/package that project reuire. But you added "type": "module" in "devDepenencies" which is not a valid package name, you need to remove this. `"type": "module" ` is a configuration that tells node how to load modules. So it will be added as a project.json top level property not inside any other property.